tools/harness-thci/README.md

```diff
@@ -1,25 +1,26 @@
-OpenThread THCI
-===============
+THCI & Test Environment Setup
+=============================
 
-OpenThread THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template "IThci",
+THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template "IThci",
 which is used by Thread Test Harness Software to control OpenThread-based reference devices according to each test
 scenario.
 
-The current implementation is based on the CC2538 example platform and is included in the current Thread Test Harness
-Software release.
+Currently, there are two THCI implementations for OpenThread. One is for OpenThread CLI which is based on the CC2538 
+example platform and is included in the current Thread Test Harness Software release. The other is for OpenThread WpanCtl
+which is based on wpantund running on a linux host (For example: Raspberry Pi 3B) working with a NCP (Network Co-Processor) 
+board (For example: Nordic Semiconductor nRF52840) running a OpenThread NCP image.
```

Comment:
nit: "For example:" -> "e.g."

Reply:
Same as earlier comment, I would suggest

(for example, Nordic Semiconductor nRF52840)